### DISCUSSION TITLE
How to copy fields from one collection to another collection

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17159] [Archive]
It would be nice if we could copy fields from collection to collection!

Example:
I have meta fields (for seo) like `meta_title`, `meta_description` and `meta_image`, which I want to use on multiple collections. At the moment I have to create them over and over again, but it would save so much time if we could just copy them. Or export and import them, or whatever.

**Comment by rijkvanzanten** [comment]
<img width="620" alt="CleanShot 2023-01-19 at 10 33 00@2x" src="https://user-images.githubusercontent.com/9141017/213484409-78f5e1aa-7bcb-453d-a3d6-2bbeb5bfd3b2.png">

Have you seen the "Duplicate Field" option? 🙂

**Reply by ghost** [reply]
@rijkvanzanten Of course, I’ve seen it ;) … what I am talking about is copying the field to another collection or to another directus instance.

**Reply by rijkvanzanten** [reply]
Duplicate field does allow you to copy it to another collection! Copying to another Directus installation is a different beast of course, as individual instances aren't aware of eachother. For that, the preferred approach is https://github.com/directus/directus/pull/14713

<img width="729" alt="CleanShot 2023-01-19 at 10 41 00@2x" src="https://user-images.githubusercontent.com/9141017/213486525-6f01b43b-4df4-446b-9b00-4176b3527ab5.png">

**Reply by ghost** [reply]
AAAAAAAAahhhhh … that made my day … haha! I didn't see it even though it was right in front of my eyes!

Thanks! 🙂
